eng/native/configurecompiler.cmake

```
@@ -112,13 +112,14 @@ if (MSVC)
 
   if (CMAKE_GENERATOR MATCHES "^Visual Studio.*$")
     # Debug build specific flags
-    # The Ninja generator doesn't appear to have the default `/INCREMENTAL:ON` that
+    # The Ninja generator doesn't appear to have the default `/INCREMENTAL:NO` that
```

The previous wording is correct.

```suggestion
    # The Ninja generator doesn't appear to have the default `/INCREMENTAL:ON` that
```

---

I don't see any `ON` option in the documentation https://learn.microsoft.com/en-us/cpp/build/reference/incremental-link-incrementally?view=msvc-170, should it just be `/INCREMENTAL` then?

---

You're right

```suggestion
    # The Ninja generator doesn't appear to have the default `/INCREMENTAL` that
```